base_tier_validation/data/cron_data.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="ir_cron_auto_tier_validation" model="ir.cron">
+        <field name="name">Automatic Tier Validation</field>
+        <field name="interval_number">1</field>
```

Comment:
This is a very high frequency, I would make it once an hour by default. You should not try to make something the closest to real-time as possible by using a very high frequency job, to do that is is better to add use queue job.

Reply:
Done